### PR TITLE
fixes for arolla

### PR DESCRIPTION
**Pull request by cosunae**
Changes required for the new version of GT

### CODE REVIEW

File: env.arolla.sh [hunk 1]
```diff
@@ -133,15 +133,15 @@ setCppEnvironment()
...
         # implicit module purge
-        module load cmake/3.14.3
         module load craype-x86-skylake
         module load craype-network-infiniband
         module load slurm
         # Gnu env
         module load PrgEnv-gnu
         module switch mvapich2/2.3.1-gcc-7.4.0-2.31.1 mvapich2/2.2-gcc-7.4.0-2.31.1-cuda-9.2
         module load cuda92/toolkit/9.2.88 craype-accel-nvidia70
-        module load cmake
```
Comment (havogt) [inline env.arolla.sh]: there seem to be no cmake now. intended?

Comment (cosunae) [reply env.arolla.sh]: it is below, line 143. Just the version provided by modules is not enough for GT, unfortunately

Comment (havogt) [reply env.arolla.sh]: Ah yes, I was just scanning for a module. Here would be one `/users/jenkins/easybuild/arolla-ln/modules/all/cmake/3.14.5`.

Comment (cosunae) [reply env.arolla.sh]: uhmm yes, I thought I tried that one, but now I see that GT requires exactly 3.14.5.... I ll try

Comment (cosunae) [reply env.arolla.sh]: done

**Pull request merged**